modular_skyrat/modules/contractor/code/datums/contractor_items.dm

```diff
@@ -218,3 +218,17 @@
 	item_icon = "arrow-left" //replace if fontawesome gets an actual hook icon
 	limited = 1
 	cost = 3
+
+/datum/contractor_item/microbomb // reset into neutral (one-way)
+	name = "Microexplosive Implant"
+	desc = "An additional subdermal microbomb implant, designed to only allow you to inject yourself. \
+	Less of a blast radius than conventional microbombs. Two of these and a standard microbomb \
+	give you the same detonating force as a microbomb. Don't give hugs. Or do. Caveat emptor."
```

Comment:
"Two of these and a standard microbomb give you the same detonating force as a microbomb" doesn't make grammatical sense